dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1603,3 +1603,18 @@ exception.occurred.while.running.ServletContainerInitializers.onStartup.useracti
 no.servlet.defined.for.servlet.filter.name.mapping=CWWWC0002W: No servlet definition is found for the {0} servlet name in the {1} filter mapping.
 no.servlet.defined.for.servlet.filter.name.mapping.explanation=The servlet name is used in filter mapping, but its definition has not been defined.
 no.servlet.defined.for.servlet.filter.name.mapping.useraction=Define the servlet which is used in the servlet name filter mapping.
+
+# {0} Cookie name that has conflict SameSite attribute
+# {1} current SameSite attribute value
+# {2} new SameSite attribute value
+duplicate.samesite.attribute.for.cookie.name=CWWWC0003W: A cookie name [{0}] with SameSite attribute value [{1}] is configured with new value [{2}].  No SameSite attribute will be added for this cookie.
+duplicate.samesite.attribute.for.cookie.name.explanation=A cookie name must not have multiple SameSite attribute values.  The attribute will not be added for the cookie.
+duplicate.samesite.attribute.for.cookie.name.useraction=The deployment descriptor's context parameters cookie names and values for 'SameSiteCookies_Lax', 'SameSiteCookies_None', and 'SameSiteCookies_Strict' must be unique across all three lists. 
+
+duplicate.reported.for.samesite.attribute.cookie.name=CWWWC0004W: A cookie name [{0}] with duplicate SameSite attribute value [{1}] was found previously.  No SameSite attribute will be added for this cookie.
+duplicate.reported.for.samesite.attribute.cookie.name.explanation=A cookie name must not have multiple SameSite attribute values.  The attribute will not be added for the cookie.
+duplicate.reported.for.samesite.attribute.cookie.name.useraction=The deployment descriptor's context parameters cookie names and values for 'SameSiteCookies_Lax', 'SameSiteCookies_None', and 'SameSiteCookies_Strict' must be unique across all three lists. 
+
+too.many.wildcard.configured.for.samesite.cookie.name=CWWWC0005W: Too many wildcard cookie name configured for SameSite attribute.  No attribute will be added for any cookie.
+too.many.wildcard.configured.for.samesite.cookie.name.explanation=Too many wildcard cookie names are configured for multiple SameSite attributes. You can configure only one wildcard cookie name across all SameSite attributes.
```

Comment:
**change**
Too many wildcard cookie names are configured for multiple SameSite attributes. You can configure only one wildcard cookie name across all SameSite attributes.

**to**
Multiple wildcard cookie names are configured for one or more SameSite attributes. You can configure only one wildcard cookie name across all SameSite attributes.